libusb/os/openbsd_usb.c

```diff
@@ -107,13 +107,19 @@ const struct usbi_os_backend usbi_backend = {
 
 	.handle_transfer_completion = obsd_handle_transfer_completion,
 
+	.get_driver_name = obs_get_driver_name,
+
 	.device_priv_size = sizeof(struct device_priv),
 	.device_handle_priv_size = sizeof(struct handle_priv),
 };
 
 #define DEVPATH	"/dev/"
 #define USBDEV	DEVPATH "usb"
 
+static const char* obs_get_driver_name(libusb_device* dev) {
+    return "usb(4)";
```

Comment:
should be a tab instead of 4 spaces

Reply:
Should it be?  libusb's indentation style is inconsistent, is there an agreed-upon style?

Reply:
look a few hundred lines up and down in this file - it is mostly tabs
that _is_ consistency, at least withing the file

introducing just one line with different indentation with contributions like this (even though it is functionally helpful) - that's what introduces inconsistencies in a first place

Reply:
> libusb's indentation style is inconsistent, is there an agreed-upon style?

The rule is follow the style of the surrounding code, or the file in question. Full consistency throughout the project is a whole different matter. They are differences e.g. between different platform-specific files, for historical reasons or preference of the major contributors. Apart from whitespace there is for example the Yoda conditions in the darwin backend :)

Reply:
@Youw oh I agree. I was asking because I had considered making a .clang-format file for the project. It could auto-format everything automatically, thus eliminating the need to nitpick patches.

Reply:
Sorry about this mistake. I should have done better. It has now been addressed. 

My experience working on this PR is that tab vs space is error prone when working on multi-platform PR. (e.g: Darwin backend uses 2 space indentation, sunOS uses 4 spaces indentation, the rest uses tabs). 

A way to avoid this could be to have a mandatory clang-format hook and bootstrap it with a global format PR.